ExampleCodes/heFFTe/Poisson/main.cpp

```diff
@@ -8,6 +8,10 @@
 
 using namespace amrex;
 
+enum struct LaplacianType {
+    Unknown, Exact, Discrete
+};
+
```

Comment:
```suggestion
AMREX_ENUM(LaplacianType,
    Unknown, Exact, Discrete
);
```
Then we can query LaplacianType directly without using string.
```
pp.query_enum_case_insensitive("laplacian_type", laplacian_type)
```

```